iml-gui/crate/src/components/action_dropdown/mod.rs

```diff
@@ -0,0 +1,367 @@
+mod confirm_action_modal;
+
+use crate::{
+    components::{attrs, dropdown, font_awesome, modal, tooltip, Placement},
+    generated::css_classes::C,
+    sleep::sleep_with_handle,
+    GMsg, MergeAttrs as _, RequestExt, WatchState,
```

Comment:
`as _` needed?

Reply:
Not needed, but also signals usage as a trait.